packages/gdl-frontend/components/LanguageMenu/LanguageMenu.js

```diff
@@ -109,7 +109,7 @@ export default class LanguageMenu extends React.Component<Props> {
                 </ListItemIcon>
                 <ListItemText inset>
                   <SrOnly>
-                    <Trans>Selected:</Trans>
+                    <Trans>Selected: </Trans>
```

Comment:
I believe we should make Trans wrap SrOnly instead perhaps. `<Trans><SrOnly>Selected: </SrOnly>{lang}</SrOnly></Trans>`